.ci/build-kit/install_and_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+cmake \
+    -B build \
+    -S "$EXT_MOUNT/source" \
```

Comment:
This line is specific for within the docker-container. I think it can be removed and the source-dir set to the current working-dir in the workflow itself.

Reply:
Is there a desire to not run the build and test in a docker container? Currently, that was the process. If there needs to be an update to that process could it be a separate issue? I would like to get this PR merged so that tests are being ran in an automated fashion.

Reply:
Yes, the scripts run in a Docker-container in GitHub (mainly that the CI could be easily debugged locally, as there no SSH-access, etc. to the CI-runners).

But this script should now also serve as the source for the local run (see https://github.com/EVerest/libocpp/pull/297#issuecomment-1849020534).

However, with the reusable workflow all this files (as all others) will be gone in this repo anyway.